src/test/fuzz/fuzz.cpp

```diff
@@ -60,9 +60,9 @@ std::map<std::string_view, std::tuple<TypeTestOneInput, TypeInitialize, TypeHidd
     return g_fuzz_targets;
 }
 
-void FuzzFrameworkRegisterTarget(std::string_view name, TypeTestOneInput target, TypeInitialize init, TypeHidden hidden)
+void FuzzFrameworkRegisterTarget(std::string_view name, TypeTestOneInput target, FuzzTargetOptions opts)
 {
-    const auto it_ins = FuzzTargets().try_emplace(name, std::move(target), std::move(init), hidden);
+    const auto it_ins = FuzzTargets().try_emplace(name, std::move(target), std::move(opts.init), opts.hidden);
```

Comment:
nit: I guess we could have `g_fuzz_targets` store the opts directly? Would mean less churn when adding an option.